src/translations/nbTranslations.json

```
@@ -454,6 +454,8 @@
         "add_publisher_claim_description": "<p>Sett eierskap til utgiver som tilhører din institusjon.</p><p>Din institusjon vil bli oppgitt som ansvarlig for resultat og kan få spørsmål via brukerstøttepunkt som angitt på institusjonens brukerstøtte.</p><p>Med å sette eierskap på utgiver så sendes publiseringsforespørsel til din institusjon uavhengig av om registrator tilhører en annen institusjon.</p>",
         "administer_publisher_channel_claim": "Administrer eierskap for utgivere",
         "administer_publisher_channel_claim_description": "<p>På denne siden kan du administrere eierskap for utgivere som tilhører din institusjon.</p><p>Eierskap settes av institusjonen, så kontakt kanaleier ved feilregistrering.</p><button/><p>Eierskap styrer hvilken institusjon som mottar publiseringsforespørselen, uavhengig av registratorens tilhørighet.</p><p>Kanaleier bestemmer om brukere fra andre institusjoner kan registrere resultater med utgiver.</p>",
+        "administer_series_channel_claim": "Administrer eierskap for serier",
```

foreslår denne navnekonvensjonen for dette
```suggestion
        "administer_serial_publication_channel_claim": "Administrer eierskap for tidsskrift/serier",
```